# SQL Part 3 Studio

### Let's practice your SQL quering skills!  For each question, work along in the notebook adding your query and answering the questions. 

### **DataSet**

### <span style="font-size: 14px; font-family: -apple-system, BlinkMacSystemFont, sans-serif;">We will be using the Goodbooks-10k dataset from the exercises in the prep work.&nbsp; Feel free reference your previous notebooks.</span>

- The dataset can be found here: [goodbooks-10k](https://www.kaggle.com/zygmunt/goodbooks-10k)
- You can access the **BooksDB** in the Launchcode server.

### **Business Issue**

You are an employee at a small independet bookstore and you have been tasked to increase sales.  You decide to use the BooksDB to find books and themes to highlight in fun promotions throughout each month / season.  We will be starting with March and then you will have a chance to explore another month of your choosing.  We want to come up with a list of events/ promotions to run each month.  If you are looking for ideas to include in your promotions:

- [https://www.calendarr.com/united-states/calendar-2022/](https://www.calendarr.com/united-states/calendar-2022/)
- [https://www.holidayinsights.com/moreholidays/](https://www.holidayinsights.com/moreholidays/)
    - Click on a month and it will take you to a more detailed page

## Part 1:  March - Women's History Month, Dr Seuss Day(3/2), National Pie Day3/14, St Patrick's Day (3/17), Season - Spring

### **Event 1:** Women's History Month

Goal: Highlight Popular Womans Authors based on ratings from the Good Reads DB.

### **Write a query that returns tag\_id, the number of times each tag\_id is used and the tag\_name.**  

### Use the GROUP BY and HAVING clause to narrow your focus.

Try multiple keywords, ie woman vs female (female ended up giving better results)

In [26]:
-- Solution
USE 
    BooksDB
SELECT 
    book_tags.tag_id AS 'TAG ID' ,tags.tag_name AS 'TAG NAME', SUM(book_tags.count) AS 'TAG COUNT'
FROM 
    book_tags
INNER JOIN 
    tags
ON 
    book_tags.tag_id=tags.tag_id
GROUP by 
    book_tags.tag_id, tags.tag_name 
HAVING 
    tag_name LIKE '%woman%' or tag_name LIKE '%female%'
ORDER BY 
    SUM(book_tags.count) DESC;




(59 rows affected)

Total execution time: 00:01:36.423

TAG ID TAG NAME TAG COUNT 11643 female-author 17999 11644 female-authors 13690 11658 female-protagonist 2554 11646 female-detective 302 11656 female-pov 155 11662 female-writers 147 32681 woman-s-murder-club 136 11642 female 133 22271 older-woman-younger-man 97 19038 man-woman 93 32740 wonder-woman 88 32685 womans-murder-club 88 22269 older-man-younger-woman 72 28834 strong-female-lead 61 11652 female-lead 61 11647 female-detectives 59 3514 author-female 47 32683 womanhood 46 28833 strong-female-characters 30 4455 biblical-womanhood 29 28830 strong-female 21 4006 batwoman 17 11650 female-friendship 14 11655 female-pi 10 11659 female-serial-killer 8 22270 older-woman 7 23671 pioneer-woman 7 28831 strong-female-character 7 937 21-books-every-woman-should-read 7 7194 christian-woman 5 5132 books-every-woman-should-read 5 28835 strong-female-protagonist 5 32680 woman-s-fiction 5 1741 afghan-female-entrepreneur-under-ta 5 19757 memorable-woman-characters 4 11653 female-main-character 3 11660 female-spy 3 11648 female-doctor 3 23891 policewoman-potential-victim 3 4729 black-ops-woman-team-cia-sean-case 3 32679 woman-author 3 11657 female-povs 2 11649 female-fiction 2 11661 female-writer 2 11645 female-biographies 2 10986 every-woman-should-read 2 32682 woman-trans-agender-authors 2 1387 a-woman-after-gods-own-heart 2 32684 womans-literature 2 1386 a-woman-after-god-s-own-heart 2 11654 female-persuasion 1 11651 female-genre-authors 1 15331 important-female-authors 1 17079 kick-ass-female 1 24455 protagonist-female 1 28832 strong-female-character-s 1 4309 best-adult-female-friendship-books 1 28836 strong-female-supporting-characters 1 29324 tbr-ax-female 1

### **Which keyword(s) and tag(s) would you like to explore more?**

murder with a tag count of 136

### **Write a query to return authors, titles, ratings and** **selected tag(s)\_id.**

In [48]:
-- Solution
USE 
    BooksDB
SELECT 
    book_tags.tag_id AS 'TAG ID', authors, books.title, books.average_rating

FROM
    book_tags
INNER JOIN 
    books
ON 
    book_tags.goodreads_book_id = books.book_id

WHERE 
    book_tags.tag_id = 11643
ORDER BY 
    books.authors, books.title, books.average_rating DESC








(997 rows affected)

Total execution time: 00:01:46.440

TAG ID authors title average_rating 11643 A.M. Homes May We Be Forgiven 3.66 11643 A.N. Roquelaure, Anne Rice Beauty's Punishment (Sleeping Beauty, #2) 3.62 11643 A.N. Roquelaure, Anne Rice Beauty's Release (Sleeping Beauty, #3) 3.71 11643 A.S. Byatt Possession 3.88 11643 A.S. Byatt The Children's Book 3.66 11643 A.S.A. Harrison The Silent Wife 3.3 11643 Adelle Waldman The Love Affairs of Nathaniel P. 3.3 11643 Agatha Christie 4:50 from Paddington (Miss Marple, #8) 3.92 11643 Agatha Christie A Caribbean Mystery (Miss Marple, #10) 3.77 11643 Agatha Christie A Murder Is Announced (Miss Marple, #5) 3.96 11643 Agatha Christie A Pocket Full of Rye (Miss Marple, #7) 3.83 11643 Agatha Christie After the Funeral (Hercule Poirot, #29) 3.84 11643 Agatha Christie And Then There Were None 4.23 11643 Agatha Christie Cat Among the Pigeons (Hercule Poirot, #32) 3.82 11643 Agatha Christie Death on the Nile (Hercule Poirot, #17) 4.06 11643 Agatha Christie Elephants Can Remember (Hercule Poirot, #37) 3.62 11643 Agatha Christie Hallowe'en Party (Hercule Poirot, #36) 3.66 11643 Agatha Christie Hercule Poirot's Christmas (Hercule Poirot, #20) 3.9 11643 Agatha Christie Murder at the Vicarage (Miss Marple, #1) 4.03 11643 Agatha Christie Murder in Mesopotamia (Hercule Poirot, #14) 3.86 11643 Agatha Christie Murder on the Orient Express (Hercule Poirot, #10) 4.16 11643 Agatha Christie Peril at End House (Hercule Poirot, #8) 3.89 11643 Agatha Christie The Big Four (Hercule Poirot, #5) 3.6 11643 Agatha Christie The Body in the Library (Miss Marple, #3) 3.85 11643 Agatha Christie The Man in the Brown Suit 3.97 11643 Agatha Christie The Moving Finger (Miss Marple, #4) 3.84 11643 Agatha Christie The Murder of Roger Ackroyd (Hercule Poirot, #4) 4.2 11643 Agatha Christie The Secret Adversary (Tommy and Tuppence #1) 3.82 11643 Agatha Christie They Do It with Mirrors (Miss Marple, #6) 3.74 11643 Agatha Christie, Robin Bailey The Clocks (Hercule Poirot, #34) 3.71 11643 Agatha Christie, Ροζίτα Σώκου The Mysterious Affair at Styles (Hercule Poirot, #1) 3.98 11643 Aimee Bender The Particular Sadness of Lemon Cake 3.19 11643 Ali Smith How to Be Both 3.61 11643 Alice Miller, Ruth Ward The Drama of the Gifted Child: The Search for the True Self 4.09 11643 Alice Hoffman Faithful 4.02 11643 Alice Hoffman Practical Magic 3.8 11643 Alice Hoffman The Ice Queen 3.51 11643 Alice McDermott Someone 3.76 11643 Alice Munro Dear Life: Stories 3.75 11643 Alice Munro Hateship, Friendship, Courtship, Loveship, Marriage: Stories 4.06 11643 Alice Munro Too Much Happiness 3.82 11643 Alice Walker Possessing the Secret of Joy 4.06 11643 Alice Walker The Color Purple 4.17 11643 Alice Waters The Art of Simple Food: Notes, Lessons, and Recipes from a Delicious Revolution 4.11 11643 Alison Bechdel Are You My Mother? 3.66 11643 Alison Bechdel Fun Home: A Family Tragicomic 4.12 11643 Alison Croggon The Naming (The Books of Pellinor, #1) 4 11643 Alison Goodman Eon: Dragoneye Reborn (Eon, #1) 3.98 11643 Alissa Nutting Tampa 3.34 11643 Allie Brosh Hyperbole and a Half: Unfortunate Situations, Flawed Coping Mechanisms, Mayhem, and Other Things That Happened 4.17 11643 Amanda Lindhout, Sara Corbett A House in the Sky 4.2 11643 Amanda Palmer The Art of Asking; or, How I Learned to Stop Worrying and Let People Help 3.97 11643 Amelia Atwater-Rhodes Hawksong (The Kiesha'ra, #1) 4.15 11643 Amy Stewart Girl Waits with Gun (Kopp Sisters, #1) 3.78 11643 Amy Poehler Yes Please 3.81 11643 Amy Schumer The Girl with the Lower Back Tattoo 3.76 11643 Amy Sedaris I Like You: Hospitality Under the Influence 3.87 11643 Amy Tan The Bonesetter's Daughter 3.98 11643 Amy Tan The Hundred Secret Senses 3.98 11643 Amy Tan The Joy Luck Club 3.9 11643 Anaïs Nin Delta of Venus 3.69 11643 Anderson Cooper, Gloria Vanderbilt The Rainbow Comes and Goes: A Mother and Son On Life, Love, and Loss 3.79 11643 Andrea Levy Small Island 3.89 11643 Angela Carter The Bloody Chamber and Other Stories 4.02 11643 Angela Flournoy The Turner 

### **Record your thoughts about why you wrote the query the way you did.**

# Part 2: Choose Another Month

Choose another month and plan at least 2 events / promotions and answer the following questions.

Which month did you choose?  

What 2 events / promotions are you highlighting?

> 1\. February: Valentine's Day

> 2\. February: Black History Month

In [49]:
USE 
    BooksDB
SELECT 
    book_tags.tag_id AS 'TAG ID' ,tags.tag_name AS 'TAG NAME', SUM(book_tags.count) AS 'TAG COUNT'
FROM 
    book_tags
INNER JOIN 
    tags
ON 
    book_tags.tag_id=tags.tag_id
GROUP by 
    book_tags.tag_id, tags.tag_name 
HAVING 
    tag_name LIKE '%love%' or tag_name LIKE '%valentine%'
ORDER BY 
    SUM(book_tags.count) DESC;

(167 rows affected)

Total execution time: 00:03:23.562

TAG ID TAG NAME TAG COUNT 18640 love 46686 18668 love-triangle 23758 18680 loved 13450 8408 cover-love 11761 12598 friends-to-lovers 8411 18682 loved-it 5756 10615 enemies-to-lovers 4132 18663 love-stories 3940 12334 forbidden-love 3901 18664 love-story 3878 18670 love-triangles 2765 18652 love-hate 1300 15646 insta-love 1228 32448 wicked-lovely 783 18653 love-hate-relationship 522 18677 lovecraftian 383 27017 second-chance-love 333 5197 books-i-love 320 18673 lovecraft 309 33148 young-love 212 14050 hate-to-love 183 32449 wicked-lovely-series 173 18667 love-the-cover 169 31578 unrequited-love 164 7674 college-love 145 13828 h-p-lovecraft 143 5200 books-i-loved-as-a-kid 137 18645 love-at-stake 128 18655 love-it 107 18656 love-love-love 84 14952 hp-lovecraft 74 18658 love-me-with-lies 69 5198 books-i-loved 59 32450 wicked-lovers 57 18688 loveozya 47 18646 love-at-stake-series 45 32438 why-men-love-bitches 40 7042 chocolate-lovers 39 25547 redeeming-love 38 19924 michelle-a-valentine 33 12041 first-love 30 18610 lost-love 26 29040 surrender-your-love 26 23236 penguin-great-loves 26 27932 slovenia 24 18641 love-and-marriage 24 18642 love-and-relationships 22 32451 wicked-lovers-series 21 19931 michelle-valentine 21 18650 love-comes-softly 20 29041 surrender-your-love-series 19 11877 fighting-for-love 18 8413 coverly-love 18 8462 crazy-love 15 27253 series-i-love 15 27115 self-love 14 15648 instalove 14 18648 love-by-numbers 13 18691 lovey-dovey 13 18654 love-hina 13 8173 cookbooks-i-love 13 18611 lost-love-series 13 18659 love-of-reading 13 18666 love-that-dog 12 15188 i-too-had-a-love-story 12 18661 love-romance 12 13775 guess-how-much-i-love-you 11 11206 fallen-in-love 11 27323 series-wicked-lovely 11 18871 mafia-love 10 17737 lawyers-in-love 10 18689 lovesick-maidens 10 18651 love-comes-softly-series 10 27272 series-love-at-stake 10 19480 maud-hart-lovelace 9 14049 hate-love 9 25024 read-and-loved 9 6164 can-love-happen-twice 9 1631 adoption-families-foster-homes-love 9 13238 girls-in-love 8 6643 charlie-lovett 8 5831 brotherly-love 8 2800 apple-love 8 2560 animal-lovers 8 2559 animal-lover 8 2080 all-love 8 18644 love-anthony 7 1343 a-love-story 7 13628 great-loves 6 10738 epic-love-story 6 18686 lovely-complex 6 4249 beloved-books 6 22207 of-course-i-love-you 5 18649 love-com 5 8743 cute-love-story 5 18657 love-marriage 5 28961 summer-love 5 5199 books-i-loved-as-a-child 5 18681 loved-as-a-kid 5 18678 lovecraftian-horror 5 1492 absolutely-loved 5 20488 most-beloved 4 22172 obsessive-love 4 22338 on-love 4 28091 someone-to-love 4 4248 beloved 4 12335 forbidden-love-affair 4 18675 lovecraft-h-p 3 6165 can-love-happens-twice 3 3749 baby-love 3 9677 dog-lover 3 18643 love-and-romance 3 17130 kids-books-i-love 3 29950 the-mastery-of-love 3 27326 seriesandauthorsilove 2 5401 books-to-love 2 6859 children-and-children-s-book-lovers 2 6896 children-s-books-i-love 2 8141 cookbook-love 2 26157 romance-love 2 12324 for-the-love-of-twilight 2 22208 of-course-i-love-you-till-i-find-so 2 15158 i-love-animals 2 4997 book-lovers-calendar 2 29861 the-i-love-books 2 20609 much-loved 2 15161 i-loved-the-book-if-you-need-a-good 2 30727 to-read-lovecraft 2 18690 lovestory 2 18660 love-relationships 2 31711 valentine-s-day 2 18662 love-sex-money 2 18669 love-triangle-it-is-complicated 2 18679 lovecraftian-mythos 2 18672 love-you-to-death 2 29988 the-ones-i-loved 1 30022 the-power-of-love 1 4722 black-love 1 4250 beloved-by-me 1 32383 what-i-love-best 1 5271 books-i-would-love-to-burn 1 28867 stuff-i-loved 1 4060 be-the-writer-you-love 1 5383 books-that-i-love 1 1100 48-days-to-the-work-you-love 1 1080 4-stars-loved-it 1 59 0-love-funny 1 6481 cee-loves 1 26201 romantic-love-stories 1 8415 covers-i-love 1 25221 read-loved 1 15159 i-love-fiction-too 1 20855 my-lovely-book-x1f49e 1 20634 multiple-lovers 1 20489 most-beloved-books 1 15160 i-love-vampire 1 15382 in-love-w-the-main-character-s 1 15647 insta-love-or-lust-baby 1 18647 love-books 1 18687 

# Part 3: Summarize your Work

For each event write at least one query that joins any two tables in the BooksDB to support your choice and record you thoughts as to why you used the paticlular query. At least one of your queries needs to include a HAVING clause.

In [50]:
-- Event 1 Query
USE 
    BooksDB
SELECT 
    book_tags.tag_id AS 'TAG ID' ,tags.tag_name AS 'TAG NAME', SUM(book_tags.count) AS 'TAG COUNT'
FROM 
    book_tags
INNER JOIN 
    tags
ON 
    book_tags.tag_id=tags.tag_id
GROUP by 
    book_tags.tag_id, tags.tag_name 
HAVING 
    tag_name LIKE '%love%' or tag_name LIKE '%valentine%'
ORDER BY 
    SUM(book_tags.count) DESC;

(167 rows affected)

Total execution time: 00:00:38.720

TAG ID TAG NAME TAG COUNT 18640 love 46686 18668 love-triangle 23758 18680 loved 13450 8408 cover-love 11761 12598 friends-to-lovers 8411 18682 loved-it 5756 10615 enemies-to-lovers 4132 18663 love-stories 3940 12334 forbidden-love 3901 18664 love-story 3878 18670 love-triangles 2765 18652 love-hate 1300 15646 insta-love 1228 32448 wicked-lovely 783 18653 love-hate-relationship 522 18677 lovecraftian 383 27017 second-chance-love 333 5197 books-i-love 320 18673 lovecraft 309 33148 young-love 212 14050 hate-to-love 183 32449 wicked-lovely-series 173 18667 love-the-cover 169 31578 unrequited-love 164 7674 college-love 145 13828 h-p-lovecraft 143 5200 books-i-loved-as-a-kid 137 18645 love-at-stake 128 18655 love-it 107 18656 love-love-love 84 14952 hp-lovecraft 74 18658 love-me-with-lies 69 5198 books-i-loved 59 32450 wicked-lovers 57 18688 loveozya 47 18646 love-at-stake-series 45 32438 why-men-love-bitches 40 7042 chocolate-lovers 39 25547 redeeming-love 38 19924 michelle-a-valentine 33 12041 first-love 30 18610 lost-love 26 29040 surrender-your-love 26 23236 penguin-great-loves 26 27932 slovenia 24 18641 love-and-marriage 24 18642 love-and-relationships 22 32451 wicked-lovers-series 21 19931 michelle-valentine 21 18650 love-comes-softly 20 29041 surrender-your-love-series 19 11877 fighting-for-love 18 8413 coverly-love 18 8462 crazy-love 15 27253 series-i-love 15 27115 self-love 14 15648 instalove 14 18648 love-by-numbers 13 18691 lovey-dovey 13 18654 love-hina 13 8173 cookbooks-i-love 13 18611 lost-love-series 13 18659 love-of-reading 13 18666 love-that-dog 12 15188 i-too-had-a-love-story 12 18661 love-romance 12 13775 guess-how-much-i-love-you 11 11206 fallen-in-love 11 27323 series-wicked-lovely 11 18871 mafia-love 10 17737 lawyers-in-love 10 18689 lovesick-maidens 10 18651 love-comes-softly-series 10 27272 series-love-at-stake 10 19480 maud-hart-lovelace 9 14049 hate-love 9 25024 read-and-loved 9 6164 can-love-happen-twice 9 1631 adoption-families-foster-homes-love 9 13238 girls-in-love 8 6643 charlie-lovett 8 5831 brotherly-love 8 2800 apple-love 8 2560 animal-lovers 8 2559 animal-lover 8 2080 all-love 8 18644 love-anthony 7 1343 a-love-story 7 13628 great-loves 6 10738 epic-love-story 6 18686 lovely-complex 6 4249 beloved-books 6 22207 of-course-i-love-you 5 18649 love-com 5 8743 cute-love-story 5 18657 love-marriage 5 28961 summer-love 5 5199 books-i-loved-as-a-child 5 18681 loved-as-a-kid 5 18678 lovecraftian-horror 5 1492 absolutely-loved 5 20488 most-beloved 4 22172 obsessive-love 4 22338 on-love 4 28091 someone-to-love 4 4248 beloved 4 12335 forbidden-love-affair 4 18675 lovecraft-h-p 3 6165 can-love-happens-twice 3 3749 baby-love 3 9677 dog-lover 3 18643 love-and-romance 3 17130 kids-books-i-love 3 29950 the-mastery-of-love 3 27326 seriesandauthorsilove 2 5401 books-to-love 2 6859 children-and-children-s-book-lovers 2 6896 children-s-books-i-love 2 8141 cookbook-love 2 26157 romance-love 2 12324 for-the-love-of-twilight 2 22208 of-course-i-love-you-till-i-find-so 2 15158 i-love-animals 2 4997 book-lovers-calendar 2 29861 the-i-love-books 2 20609 much-loved 2 15161 i-loved-the-book-if-you-need-a-good 2 30727 to-read-lovecraft 2 18690 lovestory 2 18660 love-relationships 2 31711 valentine-s-day 2 18662 love-sex-money 2 18669 love-triangle-it-is-complicated 2 18679 lovecraftian-mythos 2 18672 love-you-to-death 2 29988 the-ones-i-loved 1 30022 the-power-of-love 1 4722 black-love 1 4250 beloved-by-me 1 32383 what-i-love-best 1 5271 books-i-would-love-to-burn 1 28867 stuff-i-loved 1 4060 be-the-writer-you-love 1 5383 books-that-i-love 1 1100 48-days-to-the-work-you-love 1 1080 4-stars-loved-it 1 59 0-love-funny 1 6481 cee-loves 1 26201 romantic-love-stories 1 8415 covers-i-love 1 25221 read-loved 1 15159 i-love-fiction-too 1 20855 my-lovely-book-x1f49e 1 20634 multiple-lovers 1 20489 most-beloved-books 1 15160 i-love-vampire 1 15382 in-love-w-the-main-character-s 1 15647 insta-love-or-lust-baby 1 18647 love-books 1 18687 

### **Summary of event 1:**

Double Click to edit.

In [ ]:
-- Event 2 Query


In [56]:
USE 
    BooksDB
SELECT 
    book_tags.tag_id AS 'TAG ID' ,tags.tag_name AS 'TAG NAME', SUM(book_tags.count) AS 'TAG COUNT'
FROM 
    book_tags
INNER JOIN 
    tags
ON 
    book_tags.tag_id=tags.tag_id
GROUP by 
    book_tags.tag_id, tags.tag_name 
HAVING 
    tag_name LIKE '%black%' or tag_name LIKE '%african%' 
ORDER BY 
    SUM(book_tags.count) DESC;

(135 rows affected)

Total execution time: 00:03:23.481

TAG ID TAG NAME TAG COUNT 1753 african-american 13531 4696 black-dagger-brotherhood 7607 1750 african 1985 1761 african-american-lit 1678 1762 african-american-literature 1575 1774 african-literature 1342 4697 black-dagger-brotherhood-series 969 4695 black-dagger 889 1773 african-lit 732 14717 holly-black 633 4680 black 521 4708 black-history 417 27207 series-black-dagger-brotherhood 416 1758 african-american-fiction 391 4685 black-authors 336 4690 black-butler 304 1759 african-american-history 279 4718 black-lit 268 18406 little-black-classics 265 28185 south-african 235 1765 african-american-studies 208 1771 african-fiction 208 27499 shayla-black 182 4737 black-studies 179 4712 black-jewels 136 1754 african-american-authors 120 1767 african-authors 120 1772 african-history 116 4719 black-literature 104 23238 penguin-little-black-classics 92 4694 black-company 91 28188 south-african-literature 76 4721 black-lives-matter 71 4742 blackmail 66 29661 the-blackstone-affair 61 4716 black-library 61 4711 black-humor 61 29655 the-black-company 59 19008 malorie-blackman 55 29656 the-black-dagger-brotherhood 54 1766 african-americans 54 4724 black-magician 53 1777 african-writers 51 4749 blackstone-affair 50 4687 black-bird 50 1778 africana 47 16319 jenna-black 47 4698 black-dagger-legacy 45 29657 the-black-jewels 45 4705 black-fiction 44 29659 the-black-magician-trilogy 43 29549 terri-blackstock 42 4731 black-panthers 37 4693 black-comedy 32 4713 black-jewels-series 29 4702 black-falcon 28 4709 black-history-month 26 4701 black-experience 24 4727 black-migration-from-the-south 24 8057 contemporary-african-american-ficti 22 4736 black-stallion 21 4715 black-liberation 18 4744 blacks 17 4691 black-cat 17 4741 blackjack 17 4734 black-shayla 16 4725 black-males 16 4723 black-lung-disease 15 1764 african-american-romance 15 4717 black-list 15 4707 black-hills 14 4732 black-power 14 4743 blackness 14 4703 black-falcon-series 13 1768 african-childhood 12 4740 blackdagger-brotherhood 12 29660 the-black-stallion 10 4692 black-chick-lit 10 29703 the-chronicle-of-the-black-company 10 15981 j-r-ward-black-dagger-brotherhood 9 4739 black-white-relationships 9 1757 african-american-experience 8 12576 french-spanish-black-cultural-fusio 8 4689 black-boy 8 4714 black-jewels-trilogy 7 1769 african-culture 7 24790 r-d-blackmore 7 8658 cultural-african-american 7 4704 black-feminism 6 1776 african-studies 6 28187 south-african-lit 6 28186 south-african-fiction 6 4745 blacksmithing 6 7269 chronicles-of-the-black-company 6 6366 cassie-black 6 1770 african-diaspora 5 1775 african-slaves 5 4728 black-ops 5 29658 the-black-library 4 27945 small-black-bookcase 4 4747 blackstock-terri 4 4746 blackstock 4 4688 black-books 4 4681 black-america 4 1763 african-american-memoir 4 4733 black-romance 3 4729 black-ops-woman-team-cia-sean-case 3 4726 black-mc 3 4699 black-death 3 4684 black-author 3 4682 black-american-lit 3 13610 great-african-reads 2 4730 black-panther-party 2 4720 black-lives 2 29558 terry-blackstock 2 1760 african-american-interest 2 1756 african-american-classics 2 1752 african-america 2 29887 the-kindle-black-hole 2 4748 blackstock_terri 1 4750 blackwell-philosophy 1 6269 carefreeblackfolks 1 4738 black-urban-fiction 1 4735 black-spines 1 4722 black-love 1 20762 my-black-red-series 1 4710 black-history-read 1 4706 black-fire 1 4700 black-diamond 1 4686 black-authors-subjects 1 4683 black-and-white-photography 1 1755 african-american-books 1 1751 african-afro-american 1 1643 adult-african-american-fiction 1 97 02-black-exp 1

### **Summary of event 2:**

Double Click to edit. There are many books that would be great promotions for Black History Month, inlcuding titles and authors.